### ISSUE TITLE
Progress Bar not available.

### ISSUE THREAD

**Issue by srobledo** [GeekyAnts/native-base-docs#209]
Checking out the latest docs, looks like the Progress Bar has been removed.

**Comment by PupoSDC** [comment]
Yeah, I am also a bit confused by this. It's not available in the code base anymore? Can't find a deprecation notice anywhere...
